connector/src/main/resources/org/jboss/as/connector/subsystems/datasources/LocalDescriptions.properties

```diff
@@ -114,7 +114,7 @@ datasources.data-source.stale-connection-checker-class-name=An org.jboss.jca.ada
 datasources.data-source.stale-connection-checker-properties=The stale connection checker properties
 
 datasources.data-source.track-statements=Whether to check for unclosed statements when a connection is returned to the pool, result sets are closed, a statement is closed or return to the prepared statement cache. Valid values are: "false" - do not track statements, "true" - track statements and result sets and warn when they are not closed, "nowarn" - track statements but do not warn about them being unclosed
-datasources.data-source.transaction-isolation=Set the java.sql.Connection transaction isolation level. Valid values are: TRANSACTION_READ_UNCOMMITTED, TRANSACTION_READ_COMMITTED, TRANSACTION_REPEATABLE_READ, TRANSACTION_SERIALIZABLE and TRANSACTION_NONE
```

Comment:
In this case it's not possible If you read the updated description it says at the end: "Different values are used to set customLevel using TransactionIsolation#customeLevel.